tests/integration/_action_run_test.py

```diff
@@ -137,23 +140,26 @@ def run_action_stdout(self, **kwargs) -> Tuple[int, str, str]:
         sys.stdin = stty  # type: ignore
 
         # set stderr and stdout to fds
-        sys.stdout = tempfile.TemporaryFile()  # type: ignore
-        sys.stderr = tempfile.TemporaryFile()  # type: ignore
+        with tempfile.TemporaryFile() as sys_stdout:
```

Comment:
I think you could merge two with-blocks into one.

Reply:
good idea